content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/clientlibs/editor/js/image.js

```diff
@@ -193,7 +195,13 @@
         selector: altInputSelector,
         validate: function() {
             var seededValue = document.querySelector(altInputSelector).getAttribute("data-seeded-value");
-            var isImageFromPageImageChecked = document.querySelector('coral-checkbox[name="./imageFromPageImage"]').checked;
+            var imageFromPageImage = document.querySelector('coral-checkbox[name="./imageFromPageImage"]');
+            var isImageFromPageImageChecked;
+            if (imageFromPageImage != null) {
+                isImageFromPageImageChecked = imageFromPageImage.checked;
+            } else {
+                isImageFromPageImageChecked = false;
+            }
```

Comment:
You should be able to simply do:
```suggestion
            var isImageFromPageImageChecked = imageFromPageImage?.checked ?? false;
```

Reply:
commited, thanks.